x-pack/test/functional/apps/transform/creation_saved_search.ts

```diff
@@ -21,6 +21,7 @@ export default function ({ getService }: FtrProviderContext) {
   const transform = getService('transform');
 
   describe('creation_saved_search', function () {
+    this.tags(['pete']);
```

Comment:
Nit: remove the debug tag.

Reply:
Good spot! Done in https://github.com/elastic/kibana/pull/106990/commits/d448760791642c7b0159e2e5c46f327f989f0833